docs/identities/model/manage-identity-schema.mdx

```diff
@@ -38,15 +40,167 @@ ory patch identity-config {your-project-id} \
+1. Check the **Customize Identity Schema** box to enable editing and make the necessary changes.
+1. Enter a new name in the **Identity Model Schema** text box, for example "Customer Type 2".
+1. Click the **Update** button to save.
+
+It's recommended to manage identity schemas in version control. Learn more about
+[managing Ory Network configuration in git](http://ory.sh/docs/guides/gitops).
+
+## Update identities to use a new schema
+
+Updating the identity schema of a project can result in inconsistencies between the new schema and the identities created with the
+old schema. Follow these steps to patch identities after updating the identity schema. If you are self-hosting Ory, you can follow
+the same steps by using the API or Ory Kratos CLI.
+
+The following steps are for updating one identity. If you have more identities that should be patched to the new schema, repeat
+the steps 4 to 7 or check out the example code for bulk updating identities below.
+
+1. Retrieve the Project ID.
+
+   ```bash
+   ory list projects
+
+   export PROJECT_ID={project-id}
+   ```
+
+1. Create a new identity with the updated schema - through the registration interface or [Ory Console](https://console.ory.sh/)
+   and note down the `schema_id` and `schema_url` of the identity you just created.
+
+   ![Identity schema ID and URL](../_assets/session-information.png)
+
+1. Get all identities of the project using the following command:
+
+   ```bash
+   ory list identities --project $PROJECT_ID --format json-pretty
+   ```
+
+1. Find the identity to be updated and note down their `id`. Save the full identity to a file using the following command:
+
+   ```bash
+   export IDENTITY_ID={IDENTITY_ID}
+   ory get identity $IDENTITY_ID --project $PROJECT_ID --format json-pretty > identity-$IDENTITY_ID.json
+   ```
+
+1. Assess the required updates in traits. Remove the credential object, since you don't want to change credentials. You then need
+   to add, remove, or update the traits to match the new identity schema. You also need to change the `schema_id` and `schema_url`
+   to the new ones. For instance, adding two new traits and removing an old one:
+
+   ```diff
+   "traits": {
+   +"name": "Bob",
+   +"newsletter": true
+   -"nickname": "Bobby"
+   }
+   ```
+
+   Save the file.
+
+1. In order to update the identity, you need to use the
+   [Admin API](https://www.ory.sh/docs/reference/api#tag/identity/operation/updateIdentity). The API requires the Ory Network
+   [Project slug](https://console.ory.sh/projects/current/developers/guides) and
+   [API Key](https://console.ory.sh/projects/current/developers). Set them as environment variables:
+
+   ```bash
+   export ORY_API_KEY={api-key}
+   export ORY_SLUG={project-slug}
+   ```
+
+1. Update the identity using the following command:
+
+   ```bash
+   curl -d "@identity-$IDENTITY_ID.json" -X PUT https://$ORY_SLUG.projects.oryapis.com/admin/identities/$IDENTITY_ID \
```

Comment:
With patch you could quickly patch identities, because you only need to change the schema_id field!

https://www.ory.sh/docs/reference/api#tag/identity/operation/patchIdentity

That way, you don't need the interim step of saving the identity to disk.